deploy_docker/run.sh

```diff
@@ -1,14 +1,21 @@
 #!/bin/bash
 
-# Print command line help
-if test "$#" \< "1"; then
-  printf "\n    run.sh - runs simpleui for docker\n"
-  printf "\n    Usage: run.sh YOUR_APP_NAME\n"
-  printf "      where 'YOUR_APP_NAME' is the name of the application.\n"
-  exit 1
+
+
+all_args="$@";
+
+if test "$#" \< "2"; then
+    echo "run.sh - runs simpleui for docker";
+    echo "Usage: run.sh YOUR_APP_NAME --zmqHostname=YOUR_ZMQ_HOSTNAME";
+    echo "Other options include:";
+    printf "\t zmqHostname ==> zmq's hostname (required)\n";
```

Comment:
I would change the help text to say something like "address/hostname of native app zmq sockets", since ZMQ is really just a protocol

Reply:
fixed in last commit